infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutAlignmentDao.kt

```diff
@@ -629,68 +628,83 @@ class LayoutAlignmentDao(
         return deletedRowId
     }
 
-    fun getNodeConnectionsNear(context: LayoutContext, target: IPoint, distance: Double): List<NodeTrackConnections> =
-        getNodeConnectionsNear(context, MultiPoint(target), distance)
-
-    fun getNodeConnectionsNear(
+    fun getNodeConnectionsNearPoints(
         context: LayoutContext,
-        target: MultiPoint,
+        targets: List<MultiPoint>,
```

Comment:
Tämä metodi massoitettu raiteen uudelleenlinkityksen tehtävälistan optimoimiseksi, se kun on tällä hetkellä on ainoa paikka, missä tehdään kerralla paljon toisistaan itsenäisiä vaihdelinkityksiä.

Reply:
Hmm, olisikos tämän voinut yhdistää myös niin että olisi vain yksi iso multipoint? 

Näin puhtaasti muistin varassa ja tarkastamatta koodia, muistelisin että logiikka käsittelee jokaisen yksittäisen pisten multipointin sisältä omana noodin-yhdistely-operaationaan, joten luulisin että kaikki muutetut raiteet voisi vain niputtaa yhdeksi listaksi + kaikkien koskettujen vaihteiden pisteet yhdeksi multipointiksi ja tehdä sitten niillä yhden recalculateTopology-kutsun :thinking: 

Abstraktiona tuo recalculate on kuitenkin jotakuinkin "laske topologiat uusiksi näistä kohdista" ja raide-setti on vain muistinvaraisesti muutetut raiteet jotta laskenta ottaa huomioon muutokset joita ei ole vielä tallennettu kantaan.